Intro/README.md

```diff
@@ -0,0 +1,455 @@
+# Introduction to TPMs
+
+Trusted Platform Modules (TPMs) are a large and complex topic, made all
+the more difficult to explain by the intricate relationships between the
+relevant concepts.  This is an attempt at a simple explanation --
+simpler than reading hundreds of pages of documents, but then too, too
+light on detail to be immediately useful.
+
+So what is a TPM?  Well, it's a cryptographic co-processor with special
+features to enable "root of trust measurement" (RTM), remote attestation
+of system state, unlocking of local resources that are kept encrypted
+(e.g., filesystems), and more.  A TPM can do those things, and it can do
+it with rich authentication and authorization policies.
+
+Typically a TPM is a hardware module, a chip, though there are firmware,
+virtual, and simulated TPMs as well, all implemented in software.
+
+To simplify things we'll consider only TPM 2.0.  Also to simplify things
+we'll ignore algorithm agility.
+
+Other parts of this [tutorial](README.md) may cover specific concepts in
+much more detail.
+
+# Core Concepts
+
+Some core concepts in the world of TPMs
+
+ - cryptography:
```

Comment:
You could reduce this down to the main concepts - this is a huge list:

- TPM Objects and Hierarchies
- Keys
- NVRAM
- PCRs and Measurements
- Attestation
- Advanced Stuff
- --- SEssions, Tickets, etc

Reply:
It is a huge list, and I like that: it conveys how complex the topic is.  I don't mean to cover every item though.

Reply:
The list is a bit overwhelming to me. At least on the first reading my eyes just skip over it since "Some core concepts in the world of TPMs" doesn't convey that the list is terribly important, and I know I'll never remember 90% of it anyway.

Reply:
I pared it down a bit.